<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>consult_recordview</class>
<widget class="QWidget" name="consult_recordview">
 <property name="geometry">
  <rect>
   <x>0</x>
   <y>0</y>
   <width>1100</width>
   <height>600</height>
  </rect>
 </property>
 <property name="windowTitle">
  <string>就诊记录</string>
 </property>
 <layout class="QVBoxLayout" name="verticalLayout">
  <item>
   <layout class="QHBoxLayout" name="filterLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>就诊日期：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateFilter">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>搜索：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtSearch">
      <property name="placeholderText">
       <string>患者姓名、医生姓名或诊断</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSearch">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>科室：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbDepartment">
     </widget>
    </item>
   </layout>
  </item>
  <item>
   <layout class="QHBoxLayout" name="buttonLayout">
    <item>
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>新建记录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnView">
      <property name="text">
       <string>查看详情</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPrint">
      <property name="text">
       <string>打印记录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPrescription">
      <property name="text">
       <string>开具处方</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnExport">
      <property name="text">
       <string>导出</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStatistics">
      <property name="text">
       <string>统计</string>
      </property>
     </widget>
    </item>
   </layout>
  </item>
  <item>
   <widget class="QTableView" name="tableView">
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </item>
  <item>
   <layout class="QHBoxLayout" name="statsLayout">
    <item>
     <widget class="QLabel" name="lblTodayCount">
      <property name="text">
       <string>今日就诊：0人次</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblMonthCount">
      <property name="text">
       <string>本月就诊：0人次</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="lblCommonDiagnosis">
      <property name="text">
       <string>常见诊断：</string>
      </property>
     </widget>
    </item>
   </layout>
  </item>
 </layout>
</widget>
<resources/>
<connections/>
</ui>